src/plugins/intel_npu/CMakeLists.txt

```diff
@@ -18,6 +18,25 @@ set(NPU_PLUGIN_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
 include(cmake/features.cmake)
 
+if(MSVC AND BUILD_COMPILER_FOR_DRIVER)
```

Comment:
Better to add commends here to show we only use this new flags for release version